__tests__/app.test.js

```diff
@@ -314,3 +314,58 @@ describe("GET /api/users", () => {
       .then(({ body }) => expect(body.msg).toBe("Not Found"));
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("200: sucessfully adds a comment to the given article_id", () => {
+    return request(app)
+      .post("/api/articles/6/comments")
+      .expect(201)
+      .send({
+        username: "hello kitty",
```

Comment:
This is not a valid username so this request should actually be throwing an error.